docs/develop/api-reference/set-collections.md

```diff
@@ -117,4 +117,49 @@ Removes multiple elements from an existing set item.
 
 See [response objects](./response-objects.md) for specific information.
 
-</details>
\ No newline at end of file
+</details>
+
+### SetContainsElement
+Checks if a provided element is in the given set.
+
+| Name      | Type            | Description                                   |
+|-----------| --------------- | --------------------------------------------- |
+| cacheName | String          | Name of the cache.                            |
+| setName   | String          | Name of the set item to be altered.           |
+| element   | String \| bytes | Element to be checked its existence by this operation.   |
+
+<details>
+  <summary>Method response object</summary>
+
+The response object for SetContainsElement returns three possible options, a cache hit, miss, or an error.
+
+* Hit
```

Comment:
Below you have a nested bullet point with `containsElements(): bool[]`

I believe the Hit for this operation has a `containsElement: bool` response